src/tools/sfdk/remoteprocess.h

```diff
@@ -100,6 +100,7 @@ private slots:
     QSsh::SshConnectionParameters m_sshConnectionParams;
     QProcessEnvironment m_extraEnvironment;
     QProcess::InputChannelMode m_inputChannelMode = QProcess::ManagedInputChannel;
+    bool m_startedOk = false;
```

Comment:
I would squash the commits "Sfdk: Make RemoteProcess::m_startedOk a member variable" and "Sfdk: Allow to run device process asynchronously" together

Reply:
The former is doing refactorization prior to implementing a new feature, the latter is adding the new feature.